secant/Features/Welcome/WelcomeView.swift

```diff
@@ -12,32 +12,20 @@ struct WelcomeView: View {
     var store: WelcomeStore
         
     var body: some View {
-        GeometryReader { proxy in
-            ZStack(alignment: .top) {
-                VStack(alignment: .center, spacing: 80) {
-                    let diameter = proxy.size.width - 40
-                    ZcashBadge()
-                        .frame(
-                            width: diameter,
-                            height: diameter
-                        )
-                        .accessDebugMenuWithHiddenGesture {
```

Comment:
Please maintain the debug menu on the welcome screen since it's the only way to get there if the app fails to start up